ow_lander/src/ow_lander/actions/stow.py

```diff
@@ -0,0 +1,42 @@
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+
+import ow_lander.msg
+
+from ow_lander.server import ActionServerBase
+from ow_lander.arm import ArmActionMixin
+
+class StowServer(ArmActionMixin, ActionServerBase):
```

Comment:
You're not wrong, they are very alike. The definition of execute_action has already changed a little in the stop action branch, whic I am wrapping up now. 

I could try and experiment with doing something like this in that branch, but I am hesitant to hide away too much action implementation behind multiple levels of parent classes because at a certain point it makes seeing what the action does at a glance harder.

Reply:
In my opinion it would be more helpful to see at a glance that stow and unstow are almost identical!   Refactoring by nature often adds abstraction.

Reply:
Yea I see your point. At the risk of getting too in the weeds on this thread, all of the arm actions can be categorized into 4 general types that could all simply be different specializations of an ArmActionMixin.
1. **Moves arm direct to target joint values** - Stow and Unstow
2. **Retrieve a complex trajectory directly from trajectory_planner.py** - Grind, Dig Linear, Dig Circular, Deliver, and Discard
3. **Modifies existing joint values** - Arm Move Joint and Arm Move Joints
4. **Specialized behavior (may simply inherit ArmActionMixin directly)** - Stop and Guarded Move (which _might_ work as a **type 2** with extra bits added)

You are suggesting I make a shared base class for **type 1**. That sounds fine to me. What are your thoughts on doing this for the other types that I have listed? Keep in mind that command unification may break some of these types, and add brand new ones.

Reply:
This sounds like a good analysis and candidate for refactoring, with (1) being the easiest.  In the interest of a timely release, I'll suggest postponing this level of refactoring to after the unified commands are all implemented.